tests/Oryx.RuntimeImage.Tests/DotNetCore/DotnetCoreImageVersionsTest.cs

```diff
@@ -97,21 +99,46 @@ public void DotNetCoreBusterRuntimeImage_Contains_VersionAndCommit_Information(s
         }
 
         [Theory]
-        [InlineData("3.0", ImageTestHelperConstants.OsTypeDebianBuster, "Version: " + DotNetCoreRunTimeVersions.NetCoreApp30)]
-        [InlineData("3.1", ImageTestHelperConstants.OsTypeDebianBuster, "Version: " + DotNetCoreRunTimeVersions.NetCoreApp31)]
-        [InlineData("3.1", ImageTestHelperConstants.OsTypeDebianBullseye, "Version: " + DotNetCoreRunTimeVersions.NetCoreApp31)]
-        [InlineData("5.0", ImageTestHelperConstants.OsTypeDebianBuster, "Version: " + DotNetCoreRunTimeVersions.NetCoreApp50)]
-        [InlineData("6.0", ImageTestHelperConstants.OsTypeDebianBuster, "Version: " + DotNetCoreRunTimeVersions.NetCoreApp60)]
-        [InlineData("6.0", ImageTestHelperConstants.OsTypeDebianBullseye, "Version: " + DotNetCoreRunTimeVersions.NetCoreApp60)]
-        [InlineData("7.0", ImageTestHelperConstants.OsTypeDebianBuster, "Version: " + DotNetCoreRunTimeVersions.NetCoreApp70)]
-        [InlineData("7.0", ImageTestHelperConstants.OsTypeDebianBullseye, "Version: " + DotNetCoreRunTimeVersions.NetCoreApp70)]
+        [Trait("category", "runtime-buster")]
+        [InlineData("3.0", "Version: " + DotNetCoreRunTimeVersions.NetCoreApp30)]
+        [InlineData("3.1", "Version: " + DotNetCoreRunTimeVersions.NetCoreApp31)]
+        [InlineData("5.0", "Version: " + DotNetCoreRunTimeVersions.NetCoreApp50)]
+        [InlineData("6.0", "Version: " + DotNetCoreRunTimeVersions.NetCoreApp60)]
+        [InlineData("7.0", "Version: " + DotNetCoreRunTimeVersions.NetCoreApp70)]
         [Trait(TestConstants.Category, TestConstants.Release)]
-        public void RuntimeImage_HasExecptedDotNetVersion(string version, string osType, string expectedOutput)
+        public void RuntimeImage_Buster_HasExecptedDotNetVersion(string version, string expectedOutput)
         {
             // Arrange & Act
             var result = _dockerCli.Run(new DockerRunArguments
             {
-                ImageId = _imageHelper.GetRuntimeImage("dotnetcore", version, osType),
+                ImageId = _imageHelper.GetRuntimeImage("dotnetcore", version, ImageTestHelperConstants.OsTypeDebianBuster),
+                CommandToExecuteOnRun = "dotnet",
+                CommandArguments = new[] { "--info" }
+            });
+
+            // Assert
+            var actualOutput = string.Join("", result.StdOut.ReplaceNewLine().Where(c => !char.IsWhiteSpace(c)));
```

Comment:
I know we want this available quickly so I'm not suggesting that we fix all this tonight, but would we see any alternatives in the future to us duplicating a lot of this code? Would helper functions be a path to reduce code duplication?

Reply:
Yep we can definitely refactor this once things have settled -- we have examples of doing this across other places from previous refactors, so it's just a matter of finding some time to do it.

Reply:
In these cases, the only thing that's different per test is the Debian flavor that we're using, so we could just turn the guts of these tests into a common private method that is then called from the tests themselves, and the tests will provide the Debian flavor they want to us (along with the other arguments they're already providing, like platform version).